lib/core_ext/string/camel_case.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-require_relative "capitalize_first"
+require "core_ext/string/capitalize_first"
```

Comment:
I think all these files should be moved under `twirp/protoc_plugin` since they are fairly generic names and run a higher risk of hitting collisions with other gems that didn't scope things.

Reply:
You're right. Since installing this gem will add `lib` to the load path, there's a chance folks might accidentally get these extensions even if they never require our gem.

I'll move the `core_ext` folder, but, I think I'll keep the `google/protobuf/compiler/plugin_pb` and `google/protobuf/descriptor_pb` where they are; those are generated code from from well-known proto files for a very specific use-case.

Reply:
[cd45b12](https://github.com/collectiveidea/protoc-gen-twirp_ruby/pull/29/commits/cd45b12792664cbba83fb1cad4aff807f163ded4)